### PR TITLE
Fix osadm manage-node --schedulable

### LINKED ISSUE #10970: osadm manage-node --schedulable doesn't set Unschedulable
Attempting to mark a node as unschedulable with osadm doesn't work.  It's still possible to use 'oc edit node' and edit the node's configuration to mark it as unschedulable.

##### Version
master
##### Steps To Reproduce
    osadm manage-node openshift-master-sdn --schedulable=false

##### Current Result
    NAME                   STATUS    AGE
    openshift-master-sdn   Ready     1h

##### Expected Result

    NAME                   STATUS                     AGE
    openshift-master-sdn   Ready,SchedulingDisabled   1h

cc: @smarterclayton

### PR DESCRIPTION
**Pull request by marun**
Fixes #10970.

cc: @pravisankar

### CODE REVIEW

**Comment by marun** [comment]
[test]

File: test/cmd/admin.sh [hunk 1]
```diff
@@ -71,7 +71,8 @@ status:
 ' | oc create -f -"
 
 os::cmd::expect_success_and_text 'oadm manage-node --selector= --schedulable=true' 'Ready'
-os::cmd::expect_success_and_not_text 'oadm manage-node --selector= --schedulable=true' 'Sched'
+os::cmd::expect_success_and_not_text 'oadm manage-node --selector= --schedulable=true' 'SchedulingDisabled'
+os::cmd::expect_success_and_text 'oadm manage-node --selector= --schedulable=false' 'SchedulingDisabled'
```
Comment (smarterclayton) [inline test/cmd/admin.sh]: Add another "get" test after each of these that validates that the node is actually schedulable.  I.e. expect_succes_and_text 'oc get node ... --template="{{ .status.conditions ... }}"

Comment (stevekuznetsov) [inline test/cmd/admin.sh]: We don't have a node in `test/cmd` - that tests needs to live in e2e or extended

**Comment by pravisankar** [comment]
LGTM

**Comment by danwinship** [comment]
Networking tests still fail locally for me:

    Summarizing 7 Failures:

    [Fail] [k8s.io] Services [It] should release NodePorts on delete 
    [Fail] [k8s.io] Services [It] should create endpoints for unready pods 
    [Fail] [networking] network isolation when using a single-tenant plugin [It] should allow communication between pods in different namespaces on the same node 
    [Fail] [networking] network isolation when using a single-tenant plugin [It] should allow communication between pods in different namespaces on different nodes 
    [Fail] [k8s.io] Networking [It] should function for intra-pod communication [Conformance] 
    [Fail] [k8s.io] Networking [k8s.io] Granular Checks [It] should function for pod communication on a single node 
    [Fail] [k8s.io] Networking [k8s.io] Granular Checks [It] should function for pod communication between nodes

**Comment by marun** [comment]
@smarterclayton merge please

**Comment by smarterclayton** [comment]
You can create a node using the API in test-cmd pretty easily.  The point is to test the command, not the kubelet.

**Comment by liggitt** [comment]
Be careful about nodes in test-cmd. I feel like at one point it made the scheduler crazy to have a "Ready" node that wasn't actually ready

**Comment by marun** [comment]
Sorry, I meant kubectl, and my point stands.  The kubectl test should provide confidence that when we see 'SchedulingDisabled' that the api correctly returned 'unscheduled: true'.

**Comment by marun** [comment]
@smarterclayton How is the test you are asking for going to improve effective coverage?

**Comment by marun** [comment]
@smarterclayton afaict this fix is adequately tested.  If extra testing of this functional area is desired, please consider filing an issue and having it implemented separately rather than delaying the unblocking of the networking tests any further.

**Comment by dcbw** [comment]
flakes are https://github.com/openshift/origin/issues/11008 and https://github.com/openshift/origin/issues/9548 [test]

**Comment by dcbw** [comment]
#11008 is duped to https://github.com/openshift/origin/issues/11016; noting 11016 to get more visibility on the issue

**Comment by dcbw** [comment]
LGTM [merge]

**Comment by mfojtik** [comment]
[merge]

**Comment by danwinship** [comment]
#11016 again, [merge]

**Pull request merged**